releasenotes/notes/deprecation-warning-for-fips-proxy-configuration-5d1230917863f806.yaml

```
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+deprecations:
+  - |
+    Deprecation warnings added for the fips-proxy configuration keys (eg. `fips.*`)
+    as this is planned to be unsupported in 7.65+ releases of the `datadog-agent`.
+    Please use the `datadog-fips-agent` instead.
```

```suggestion
    To continue using the FIPS proxy, use the `datadog-fips-agent` instead.
```
i think this is what's being implied here

---

clarified this in https://github.com/DataDog/datadog-agent/pull/34057/commits/3554f492c68153cf521ffd95020b06debb14b0e3